spec-cordova/platform/addHelper.spec.js

```diff
@@ -0,0 +1,515 @@
+/**
```

Comment:
Sort of. The old tests I `xdescribe`d them so they don't run. Instead, based on the code and general purpose of the `addHelper` module, I wrote up four `it()` statements describing tests we should implement to get decent coverage and exercise the spirit of the module. I also added a big mulitline comment roughly sketching out what `addHelper` does - we should convert that comment into more `it()` statements to lay out the overview of unit tests for this module.

Finally, we should go over the old tests I `xdescribe`d at the bottom and figure out a) if we already have the coverage for what those tests do elsewhere, or if not b) write up unit tests to replace them (possibly in a different module - depends on what the test is testing).